$NetBSD: patch-src_webhttrack,v 1.1 2013/07/09 14:10:19 noud4 Exp $

make html dir (containing server pages) findable.
difference between ps option on Linux or NetBSD.

--- src/webhttrack.orig	2013-07-09 13:19:58.000000000 +0000
+++ src/webhttrack
@@ -17,9 +17,9 @@ if test "`uname -s`" == "Darwin"; then
 BROWSEREXE="/usr/bin/open -W"
 fi
 BINWD=`dirname "$0"`
 SRCHPATH="$BINWD /usr/local/bin /usr/share/bin /usr/bin /usr/lib/httrack /usr/local/lib/httrack /usr/local/share/httrack /opt/local/bin /sw/bin ${HOME}/usr/bin ${HOME}/bin"
 SRCHPATH="$SRCHPATH "`echo $PATH | tr ":" " "`
-SRCHDISTPATH="$BINWD/../share $BINWD/.. /usr/share /usr/local /usr /local /usr/local/share ${HOME}/usr ${HOME}/usr/share /opt/local/share /sw ${HOME}/usr/local ${HOME}/usr/share"
+SRCHDISTPATH="$BINWD/../share $BINWD/.. /usr/share /usr/local /usr /local /usr/local/share/httrack /usr/local/share ${HOME}/usr ${HOME}/usr/share /opt/local/share /sw ${HOME}/usr/local ${HOME}/usr/share"
 
 ###
 # And now some famous cuisine
@@ -55,7 +55,12 @@ if ! test -n "${user_name}"; then
 user_name=`id -un`
 fi
 if test -n "${user_name}"; then
-ps -e -U "$user_name" | grep -E "(iceape|mozilla|netscape|firebird|firefox)" | grep -qv "grep -E"
+if test "`uname -s`" == "NetBSD"; then
+PSALL="-a"
+else
+PSALL="-e"
+fi
+ps ${PSALL} -U "$user_name" | grep -E "(iceape|mozilla|netscape|firebird|firefox|xulrunner)" | grep -qv "grep -E"
 else
 false
 fi
